src/app/app-routing.module.ts

```diff
@@ -0,0 +1,14 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+import { CoursesComponent } from './courses/courses.component';
+
+const routes: Routes = [
+  { path: 'courses', component: CoursesComponent },
+  { path: '',   redirectTo: '/courses', pathMatch: 'full' }
```

Comment:
please, move it on 6th line, so that root url will be at the top pf routes definition.

Reply:
done